### PR TITLE
add check : PrefixSum.get_sum start should be less or equal to the end.

### PR DESCRIPTION
**Pull request by alexpantyukhin**
### Describe your change:

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by zhudingsuifeng** [comment]
您好，邮件已经收到，谢谢

**Comment by BCGL** [comment]
您好，邮件已收到！

**Comment by wwowowo** [comment]
您好，您的邮件已经收到。我会尽快处理，谢谢！

File: data_structures/arrays/prefix_sum.py [hunk 1]
```diff
@@ -34,7 +34,15 @@ def get_sum(self, start: int, end: int) -> int:
...
         ...
         IndexError: list index out of range
+        >>> PrefixSum([1,2,3]).get_sum(2, 1)
+        Traceback (most recent call last):
+        ...
+        ValueError: start should be less or equal than end
         """
+
+        if start > end:
+            raise ValueError("start should be less or equal than end")
```
Comment (CaedenPH) [inline data_structures/arrays/prefix_sum.py:44]:
```suggestion
            raise ValueError("Start cannot be greater than end")
```

**Review by CaedenPH — APPROVED** [review]

**Pull request closed without merging**